textract/cli.py

```diff
@@ -61,6 +61,10 @@ def get_parser():
         help='Specify the encoding of the output.',
     )
     parser.add_argument(
+        '--extension', type=str, default='',
+        help='Specify the extension of the file.',
```

Comment:
It would be good to also give some examples in the `help` text. Perhaps something like "Specify the extension of the file (e.g., '.docx' or '.pdf')".

I personally think it would also be nice to accept the extension without the leading `.`, in which case the `help` text might read "Specify the extension of the file (e.g., '.docx' or 'docx')" to make it clear that both types are possible.